tests/lib/naming_test.py

```diff
@@ -169,5 +171,123 @@ def testParseServiceFile(self):
     filedefs._ParseFile(data, 'services')
     self.assertEqual(filedefs.GetService('HTTP'), ['80/tcp'])
 
+
+class Naming_Characterization_Tests(unittest.TestCase):
```

Comment:
Hm, not sure!  When I added these tests, there weren't any unit tests.  I was creating a yaml version of this class, and needed to ensure that the behaviour was consistent.  I couldn't figure out what the behaviour was without running it, and figured that I may as well commit this exploration in the form of tests for the overall project health.

Also, the naming.py class looked like it could use some refactoring, so these tests would make that safe.  "If it ain't broke, don't fix it" notwithstanding, I think there may be some style issues too ... really wasn't sure.